### PR TITLE
Option to redirect output and keep it on exit.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

File: swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt [hunk 1]
```diff
@@ -299,8 +299,12 @@ private fun waitForEmulatorToFinishBoot(targetEmulator: Emulator): Observable<Em
...
 private fun Long.nanosAsSeconds(): Float = NANOSECONDS.toMillis(this) / 1000f
 
-private fun outputFileForEmulator(args: Commands.Start) = File("${args.emulatorName}.output")
-        .apply { deleteOnExit() }
+private fun outputFileForEmulator(args: Commands.Start) =
+        File(
+                args.redirectOutputTo ?: "",
+                "${args.emulatorName}.output"
+        )
+                .apply { if (!args.keepOutputOnExit) deleteOnExit() }
```
Comment (ming13) [inline swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Formatting seems wrong.

Comment (yunikkk) [reply swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Hmm, you mean the indent?

Comment (ming13) [reply swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]:
```kotlin
private fun outputFileForEmulator(args: Commands.Start) = File(args.redirectOutputTo ?: "", "${args.emulatorName}.output").apply {
    if (!args.keepOutputOnExit) deleteOnExit()
}
```
or
```kotlin
private fun outputFileForEmulator(args: Commands.Start) =
    File(args.redirectOutputTo ?: "", "${args.emulatorName}.output").apply {
        if (!args.keepOutputOnExit) deleteOnExit()
    }
```
Not sure what auto-formatter will do.

Comment (yunikkk) [reply swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Ok with 2 one

Comment (dmitry-novikov) [reply swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt]: Another one:
```kotlin
private fun outputFileForEmulator(args: Commands.Start) =
        File(
                args.redirectOutputTo ?: "",
                "${args.emulatorName}.output"
        ).apply {
            if (!args.keepOutputOnExit) deleteOnExit()
        }
```

**Comment by artem-zinnatullin** [comment]
Niice, thanks!

**Pull request merged**